components/org.wso2.carbon.identity.conditional.auth.functions.analytics/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/analytics/internal/AnalyticsFunctionsServiceComponent.java

```diff
@@ -103,6 +103,44 @@ protected void activate(ComponentContext context) {
         }
     }
 
+    @Activate
+    protected void activate(ComponentContext context) {
```

Comment:
Consider adding a log statement here to indicate that the analytics functions service component is being activated.